packages/web-components/src/button/button.styles.ts

```diff
@@ -279,37 +283,33 @@ export const baseButtonStyles = css`
 export const styles = css`
   ${baseButtonStyles}
 
-  :host(:is([disabled], [disabled-focusable], [appearance][disabled], [appearance][disabled-focusable])),
-  :host(:is([disabled], [disabled-focusable], [appearance][disabled], [appearance][disabled-focusable]):hover),
-  :host(:is([disabled], [disabled-focusable], [appearance][disabled], [appearance][disabled-focusable]):hover:active) {
+  :host(:is(:disabled, [disabled-focusable], [appearance]:disabled, [appearance][disabled-focusable])),
```

Comment:
Some of these `disabled`-related selectors seem repetitive - could they be cleaned up with a `const` as well?

Reply:
I want to take a pass at this to see how much specificity we need across these after the fact and what all we need selector-wise.